core/src/main/java/org/apache/accumulo/core/fate/zookeeper/MetaFateStore.java

```diff
@@ -223,7 +223,7 @@ private void seedTransaction(Fate.FateOperation txName, Repo<T> repo, boolean au
     if (autoCleanUp) {
       txStore.setTransactionInfo(TxInfo.AUTO_CLEAN, autoCleanUp);
     }
-    txStore.setTransactionInfo(TxInfo.TX_NAME, txName);
+    txStore.setTransactionInfo(TxInfo.TX_NAME, fateOp);
```

Comment:
ensure that TxInfo.TX_NAME is no longer referenced as it no longer exists. This, any maybe elsewhere, needs to be changed to
`TxInfo.FATE_OP`